mod_pbxproj/mod_pbxproj.py

```diff
@@ -725,19 +727,21 @@ def remove_single_valued_flag(self, flag, configuration='All'):
             if b.remove_single_valued_flag(flag):
                 self.modified = True
 
-    def add_embed_binaries(self, embed_binaries):
+
+    def add_embed_binaries(self,embed_binaries,pbx_native_target='Unity-iPhone'):
         if len(embed_binaries) > 0:
             self.modified = True
             self.add_single_valued_flag("LD_RUNPATH_SEARCH_PATHS", "$(inherited) @executable_path/Frameworks", "Release")
             self.add_single_valued_flag("LD_RUNPATH_SEARCH_PATHS", "$(inherited) @executable_path/Frameworks", "Debug")
             for binary in embed_binaries:
-                self.add_embed_framework(binary)
+                self.add_embed_framework(pbx_native_target,binary)
 
-    def add_embed_framework(self,file_name):
+    #"Unity-iPhone"
+    def add_embed_framework(self,pbx_native_target,file_name):
```

Comment:
keeping the order will improve backwards compatibility. Please change the order of the parameters and define the pbx_native_target with default 'Unity-iPhone'